react/components/InfoCard/index.js

```diff
@@ -1,12 +1,13 @@
 import classNames from 'classnames'
-import insane from 'insane'
 import { bool, string, oneOf } from 'prop-types'
+import React, { memo, useState, useEffect, useRef } from 'react'
+import { Renderer } from 'marked'
 import { values } from 'ramda'
-import React, { memo, useMemo } from 'react'
 import { injectIntl, intlShape } from 'react-intl'
 import { useRuntime } from 'vtex.render-runtime'
 import { formatIOMessage } from 'vtex.native-types'
 import { useCssHandles } from 'vtex.css-handles'
+import RichText from 'vtex.rich-text/index'
```

Comment:
```suggestion
import RichText from 'vtex.rich-text'
```

Reply:
Hello @kaisermann, by removing the `/index` on `RichText` import will cause this error: https://i.imgur.com/h4UBjoJ.png

Reply:
Hi @marcosewbank, did you try doing this?
```suggestion
import { RichText } from 'vtex.rich-text'
```

Reply:
Yes @Klynger, i've tryed but it didn't work :cry: 
the error: https://i.imgur.com/8XfqNZL.png
btw, thank you for the help :smile: